96-chromebook.install

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/bash
```

Comment:
Can you prepend de commit message title with `chromebook-setup:`

`chromebook-setup: Extend Fedora to support kernel updates on Chromebooks`

Just to follow the same pattern used in all other commits.

Reply:
ok will do thanks